apps/manifest_test.go

```diff
@@ -1,155 +1,167 @@
-package apps_test
```

Comment:
I quite like the back box as it ensure to only test the API. Does it make sense to move the migration code into a separate file?